appium/tests/specs/live/inbox/CheckReplyMessageWithChangingRecipient.spec.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  SplashScreen,
+  SetupKeyScreen,
+  MailFolderScreen,
+  EmailScreen,
+  NewMessageScreen,
+  SearchScreen
+} from '../../../screenobjects/all-screens';
+
+import { CommonData } from '../../../data';
+
+describe('INBOX: ', () => {
+
+  it('test for replying to my own email and changing recipient', async () => {
+
+    const senderEmail = CommonData.emailForReplyWithChangingRecipient.senderEmail;
+    const oldRecipientName = CommonData.emailForReplyWithChangingRecipient.recipientName;
+    const emailSubject = CommonData.emailForReplyWithChangingRecipient.subject;
+    const secondMessage = CommonData.emailForReplyWithChangingRecipient.secondMessage;
+    const newRecipientEmail = CommonData.emailForReplyWithChangingRecipient.newRecipientEmail;
+    const newRecipientName = CommonData.emailForReplyWithChangingRecipient.newRecipientName;
+
+    const replySubject = `Re: Re: ${emailSubject}`;
+    const quoteText = `${senderEmail} wrote:\n > ${secondMessage}`;
+
+    await SplashScreen.login();
+    await SetupKeyScreen.setPassPhrase();
+    await MailFolderScreen.checkInboxScreen();
+
+    await MailFolderScreen.clickSearchButton();
+    await SearchScreen.searchAndClickEmailBySubject(emailSubject);
+    await EmailScreen.checkThreadMessage(senderEmail, emailSubject, secondMessage, 1);
+
+    // check reply message
+    await EmailScreen.clickReplyButton();
+    await NewMessageScreen.checkFilledComposeEmailInfo({
+      recipients: [oldRecipientName],
+      subject: replySubject,
+      message: quoteText
+    });
```

Comment:
Please try this slightly different: the original message should have `to:recipient1,recipient2` and `cc:recipient3`. That will make it easier to test the updated implementation.

Reply:
recipient1  -  e2e account
recipient3 - demo account
recipient2 - my flowcrypt account
@tomholub is it correct?
I will send the email from my account (dmitry@flowcrypt.com) to cc(recipient3) and to recipient1

Reply:
or the 2nd recipient should be robot@flowcrypt.com?

Reply:
The point is, the original message that you are replying to should have more than one recipient, as described above. You can use demo, robot, your account for example, details don't matter.

Then we will observe which of the recipients of the original message were rendered in recipients once you tap to reply-all.

Reply:
> if i click on reply all button i see all recipients

that one is correct

Reply:
and what "reply" should be fixed?

Reply:
#1590 may have fixed this at least partially, remaining discussion will be in https://github.com/FlowCrypt/flowcrypt-ios/issues/1599 , in the meanwhile please test for the above as you have described it to me, and fix this comment https://github.com/FlowCrypt/flowcrypt-ios/pull/1582#discussion_r868294123 and we can merge this test.

I'll mark this PR as draft. Ensure you are on latest master, and once this PR is ready, mark it back to ready so that we can review.

Reply:
@fcvakintos Please merge master into this branch.
Master branch already contains fix

Reply:
It should not display `e2e.enterprise.test` as `To` recipient in master branch.

Reply:
I don't see the merge commit that updates this PR from master. Are you sure?

Reply:
As I mentioned, e2e.enterprise.test should not be be listed in `To` recipient when reply.
Please double check if you are on latest master changes.

Reply:
![image](https://user-images.githubusercontent.com/77344191/168115290-2bea3075-e1c9-44f4-9963-9383e414d745.png)

Reply:
checked now, works, but I dont see the cc recipient after click on Reply icon 
<img width="397" alt="Screenshot 2022-05-13 at 13 24 58" src="https://user-images.githubusercontent.com/91205853/168264981-d10310cd-6585-40ef-80dc-483526389916.png">

Reply:
@ioanmo226 ^

Reply:
I pushed fix for test

Reply:
I can see that this branch is now upto date with master.
So I suggest to resolve this conversation and start new one if new problem occurs.
It's hard to track what's missing and what's not missing here as conversation is too long.